collections/sentinel-5p-l2-tropomi.yaml

```diff
@@ -29,7 +29,8 @@ Tags:
   - open data
   - sentinel
 License: |
-  [License](https://docs.sentinel-hub.com/api/latest/data/sentinel-5p-l2/#attribution-and-use)
+  - <a href="https://www.sentinel-hub.com/tou/" target="_blank">Sentinel Hub Terms of Use</a>.
+  - <a href="https://docs.sentinel-hub.com/api/latest/data/sentinel-5p-l2/#attribution-and-use" target="_blank">Attribution and Use </a>.
```

Comment:
Attribution and Use > Sentinel-5P Attribution and Use

Reply:
done